pkgs/applications/misc/direwolf/default.nix

```diff
@@ -1,38 +1,41 @@
-{ stdenv, fetchFromGitHub, unzip, alsaLib }:
-let
-  version = "1.2";
-in
+{ stdenv, fetchFromGitHub
+, espeak, alsaLib, perl
+, python }:
+
 stdenv.mkDerivation rec {
   name = "direwolf-${version}";
-  inherit version;
+  version = "1.3";
 
   src = fetchFromGitHub {
     owner = "wb2osz";
     repo = "direwolf";
-    rev = "8b81a32";
-    sha256 = "0r4fgdxghh292bzhqshr7zl5cg2lfsvlgmy4d5vqcli7x6qa1gcs";
+    rev = version;
+    sha256 = "1x6vvl3fy70ic5pqvqsyr0bkqwim8m9jaqnm5ls8z8i66rwq23fg";
   };
 
   buildInputs = [
-    unzip alsaLib
+    alsaLib espeak perl python
```

Comment:
`alsaLib` is a linux only dependency. But it should work fine on other platforms without it, using something like this:

```
] ++ stdenv.lib.optional stdenv.isLinux alsaLib;
```